<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-8T676KP" ignored="0" name="ActiTime" tests="1" failures="1" timestamp="2020-08-04T01:53:16 IST" time="54.51" errors="0">
  <testcase name="invalidLoginTest" time="0.002" classname="com.actitime.timetrack.ActiInvalidLogin">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.actitime.timetrack.ActiInvalidLogin.invalidLoginTest(ActiInvalidLogin.java:16)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- invalidLoginTest -->
</testsuite> <!-- ActiTime -->
